### PR TITLE
Correct %lp macro implementation

### PR DESCRIPTION
**Pull request by df-ferital**
This macro is used to get the current province name (reverse-engineered from classic). It is only used in TEXT.RSC record 8375, which corresponds to Mid Year 16th celebration text.

### CODE REVIEW

**Comment by Interkarma** [comment]
Nice work! :)

**Comment by jotoho** [comment]
@Interkarma 
Are you sure that you wanted to merge master into this and not the other way around?

Just want to make sure, since it seems a bit strange...

File: Assets/Scripts/Internal/PlayerGPS.cs [hunk 1]
```diff
@@ -385,27 +385,10 @@ public NameHelper.BankTypes GetNameBankOfCurrentRegion()
...
-            {
-                case FactionFile.FactionRaces.Redguard:
-                    return Races.Redguard;
-                case FactionFile.FactionRaces.Nord:
-                    return Races.Nord;
-                case FactionFile.FactionRaces.Breton:
-                default:
-                    return Races.Breton;
-            }
+            return (Races) MapsFile.RegionRaces[GameManager.Instance.PlayerGPS.CurrentRegionIndex] + 1;
```
Comment (Interkarma) [inline Assets/Scripts/Internal/PlayerGPS.cs:391]: I don't agree with this change. Classic definitely generates Nord NPCs around those subclimates, and RegionRaces array only defines Breton or Redguard (0 or 1). This change would remove that diversity and will not match classic. For example, NPC selection in Wrothgarion Mountains/Whitetry in classic and DFU with existing code will match:

**Classic**
fall_102


**DFU**
image


And then after this code change the NPCs no longer match.

**After Edit**
image


Happy to take the %lp macro, just not with this edit included. Perhaps we need to rethink the accuracy of MapsFile.RegionRaces instead.

**Comment by Interkarma** [comment]
> @Interkarma
> Are you sure that you wanted to merge master into this and not the other way around?
> 
> Just want to make sure, since it seems a bit strange...

The PR was out of date and referenced code that no longer exists in master (HardStrings). It also had a conflict that had to be resolved. What you're seeing is the result of making those edits and resolving conflict.

**Pull request merged**